.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       run: |
           sudo apt-get -q update
           sudo -E apt-get -yq --no-install-suggests --no-install-recommends install python2-minimal
-          cd /tmp && go get -u github.com/vbatts/git-validation
+          cd /tmp && go install github.com/vbatts/git-validation@latest
```

Comment:
FWIW; this one was already included in https://github.com/distribution/distribution/commit/2392893bcfe7de3010c66626db46f59ab520c9a0, so should dissolve when rebasing.